pcsx2/gui/Panels/CpuPanel.cpp

```diff
@@ -171,6 +171,7 @@ Panels::CpuPanelEE::CpuPanelEE( wxWindow* parent )
 	*this += m_button_RestoreDefaults | StdButton();
 
 	Connect( wxID_DEFAULT, wxEVT_COMMAND_BUTTON_CLICKED, wxCommandEventHandler( CpuPanelEE::OnRestoreDefaults ) );
+	Connect( wxID_ANY, wxEVT_COMMAND_RADIOBUTTON_SELECTED, wxCommandEventHandler(CpuPanelEE::EECache_Event) );
```

Comment:
(Optional)

The wxWidgets 3.0 documentation recommends using `Bind()` instead of `Connect()`. The equivalent line would be
`Bind(wxEVT_COMMAND_RADIOBUTTON_SELECTED, &CpuPanelEE::EECache_Event, this);`